vllm/v1/worker/gpu_model_runner.py

```diff
@@ -161,14 +161,18 @@ def __init__(
 
         # Set up speculative decoding.
         self.use_spec_decode = False
+        self.use_aux_hidden_state_outputs = False
```

Comment:
nit: Can we rename this?
```suggestion
        self.use_eagle3 = False
```

Reply:
I think it makes sense to have this functionality distinct from EAGLE3. For now it is the only method that needs hidden state outputs, but in the future there may be other methods which use them in a similar way. I would prefer to keep the `aux_hidden_states_outputs` the same and have eagle-3 be a configuration of that option.

If you feel strongly about this, I can change it.

Reply:
Ok. I don't have a strong preference either, so let's keep it and refactor it when needed. :)